lib/std/os/linux.zig

```diff
@@ -4532,8 +4532,20 @@ pub const clockid_t = enum(u32) {
     BOOTTIME = 7,
     REALTIME_ALARM = 8,
     BOOTTIME_ALARM = 9,
-    SGI_CYCLE = 10,
-    TAI = 11,
+    // In the linux kernel header file (time.h) is the following note:
+    // * The driver implementing this got removed. The clock ID is kept as a
+    // * place holder. Do not reuse!
+    // Therefore, calling clock_gettime() with these IDs will result in an error.
+    // SGI_CYCLE = 10,
```

Comment:
Technically these could be useful if someone wants to compile a Zig program to run against an old Linux kernel that supports these old clocks?  (I suspect that's really unlikely in this case though?)  Anyway, it seems a bit odd for Zig to only track what is currently supported, so maybe its just sufficient to add a comment that these clocks are not supported on modern systems?  
```
    SGI_CYCLE = 10, // Linux dropped support for this clock in the v4.13 timeframe
    TAI = 11, // Linux dropped support for this clock in the v4.13 timeframe
```

I may be missing some context on these clocks, so feel free to ignore this if its not compelling to you.

Reply:
TAI_CLOCK was designed as CLOCK_REALTIME(UTC) + tai_offset, but tai_offset was always 0. So there is no point in using this clock, and no one did. 
SGI_CYCLE was for Silicon Graphics (SGI) workstations, which are probably no longer in use, so it makes sense to disable these two.